<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1073</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Handball Manager</string>
  </property>
  <widget class="QLabel" name="lblTitle">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>10</y>
     <width>301</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>32</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Contract Offers</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblClub">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>301</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Club: </string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tblPlayers">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>119</y>
     <width>601</width>
     <height>841</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Player</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Age</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Position</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Attack</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Defense</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Club</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Selected</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Salary</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Length</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btnNext">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>850</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Next Club</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConfirm">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>770</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblBudget">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>80</y>
     <width>131</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Budget: </string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPlayers">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>80</y>
     <width>131</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Players:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
